front/app/modules/commercial/project_folders/citizen/containers/ProjectFolderShowPage/ProjectFolderDescription.tsx

```diff
@@ -217,6 +214,11 @@ const ProjectFolderDescription = memo<Props & InjectedIntlProps>(
                 </CollapseButtonWrapper>
               )}
           </Description>
+          {!isNilOrError(projectFolderFiles) &&
```

Comment:
Same here, the links to project files (e.g. .pdf files) could have been hidden but not fixed by our new `disableTabbing` function, so it wouldn't have worked with our fix. Now it's always visible, so no issues with the hidden content should appear.

Reply:
Makes sense 👍